docs/debug.md

```diff
@@ -0,0 +1,150 @@
+# Debugging SOCI snapshotter
+
+This document outlines where to find/access logs and metrics for the snapshotter. It attempts to provide some common error paths that a user might run into while using the snapshotter and provides some guidance on how to either root-cause the issue or resolve it.
+- [Debugging SOCI snapshotter](#debugging-soci-snapshotter)
+  - [Finding Logs / Metrics](#finding-logs--metrics)
+    - [Logs](#logs)
+    - [Metrics](#metrics)
+      - [Accessing Metrics](#accessing-metrics)
+      - [Metrics Emitted](#metrics-emitted)
+  - [Common Scenarios](#common-scenarios)
+    - [Rpull](#rpull)
+      - [No lazy-loading](#no-lazy-loading)
+      - [Pull taking an abnormal amount of time](#pull-taking-an-abnormal-amount-of-time)
+      - [Background Fetching](#background-fetching)
+    - [Running container](#running-container)
+      - [FUSE Read Failures](#fuse-read-failures)
+
+
+
+## Finding Logs / Metrics
+
+### Logs
+
+Before debugging it is important to first establish what logs would be most useful and where they can be located. For the most part, the SOCI grpc logs will be the most important place to look when debugging. If the SOCI snapshotter grpc server was started via `systemd` than you can obtain logs using `journalctl`. 
+
+`journalctl -u soci-snapshotter-grpc.unit`
+
+If you have started SOCI-snapshotter grpc manually, logs will either be emitted to stderr/stdout or to the destination of your choice.
+
+### Metrics
+
+#### Accessing Metrics
```

Comment:
this header 4 seems redundant. header 2 is `## Finding Logs / Metrics` and this part is that content for metrics (`### Metrics`)

Reply:
I think have a TOC to jump to what metrics are emitted / how to find them might be valuable. But I'm fine with removing it.

Reply:
it's a nit, so either way is fine to me :)